TTS-Common-Control-Policy.md

```diff
@@ -24,12 +24,12 @@ The GSA Chief Information Security Officer is ultimately responsible for these p
 
 ## Coordination 
 
-GSA Information Security works in close coordination and collaboration with all of the Services of the GSA, especially the [Technology Transformation Service (TTS)](https://www.gsa.gov/tts). 18F, an office of TTS, is home to multiple infrastructure and security experts. Implementations, and recommendations for policy additions and improvements, are constantly coordinated between the two teams. 
+GSA Information Security works in close coordination and collaboration with all of the Services of the GSA, especially the [Technology Transformation Service (TTS)](https://www.gsa.gov/tts). TTS Solutions, an office of TTS, is home to multiple infrastructure and security experts. Implementations, and recommendations for policy additions and improvements, are constantly coordinated between the two teams. 
 
 ## Compliance
 
-The TTS Infrastructure Director is ultimately responsible for ensuring that the procedures of all TTS systems, including cloud.gov, adhere to these policies. The TTS Infrastructure Director accepts the risk of operating all Low and Moderate impact systems as the Authorizing Official, and reviews for compliance before granting an authority to operate (ATO).	
+The TTS Technology Portfolio Director is ultimately responsible for ensuring that the procedures of all TTS systems, including cloud.gov, adhere to these policies. The TTS Technology Portfolio Director accepts the risk of operating all Low and Moderate impact systems as the Authorizing Official, and reviews for compliance before granting an authority to operate (ATO).
 
 ## Reviews
 
-GSA Information Security and TTS Infrastructure reviews all policies and procedures when new ATOs are required, or once every 3 years, whichever comes first. The teams also review modifications to policies and procedures when modifications become necessary due to changes in law, regulation, or operations.
\ No newline at end of file
+GSA Information Security and TTS Technology Portfolio reviews all policies and procedures when new ATOs are required, or once every 3 years, whichever comes first. The teams also review modifications to policies and procedures when modifications become necessary due to changes in law, regulation, or operations.
```

Comment:
```suggestion
GSA Information Security and the TTS Technology Portfolio review all policies and procedures when new ATOs are required, or once every 3 years, whichever comes first. The teams also review modifications to policies and procedures when modifications become necessary due to changes in law, regulation, or operations.
```

Honestly, as a JAB system, cloud.gov is a bit of an outlier, and doesn't operate the same way as other GSA systems authorizations-wise. While true for other systems, not sure this section applies to cloud.gov.